File: programming/robot/src/main/java/robotics/Location.java
package robotics;

public class Location {
    int x;
    int y;
    Location(int x, int y) {
        this.x = x;
        this.y = y;
    }
    int GetX() {
        return x;
    }   
    int GetY() {
        return y;
    }   
    void SetX(int x) {
        this.x = x;
    }
    void SetY(int y) {
        this.y = y;
    }
}

File: programming/robot/src/Location.cpp
#include "Location.h"

Location::Location(int x_val, int y_val)
{
    x = x_val;
    y = y_val;
}

int Location::GetX() const
{
    return x;
}

int Location::GetY() const
{
    return y;
}

void Location::SetX(int x_val)
{
    x = x_val;
}

void Location::SetY(int y_val)
{
    y = y_val;
}